design/CreateExperiment.md

```diff
@@ -5,29 +5,73 @@ tuned.
 
 # Attributes
 
-* experiment_name \
+### Experiment
+- **version** \
+  Signifies the API version.
+- **experiment_name** \
   A unique string name is specified for identifying individual experiments.
-* deployment_name \
-  Stay tuned
-* namespace \
-  Stay tuned
-* performance_profile \
-  `performance_profile` is a pre-defined performance objective that is baked into Kruize.
+- **cluster_name** \
+  Name of the cluster where the app is deployed.
 
+### Performance Objective
+- **performance_profile** \
+    `performance_profile` is a pre-defined performance objective that is baked into Kruize. 
+- **slo** \
+  Service Level Objective to be defined in the `performance_profile` as shown in [Performance Profile](https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_openshift.yaml) section, or it can be defined here in the experiment JSON and a Default Performance Profile will be created. 
```

Comment:
This is not the correct description. Please add more details here to explain SLO better. See the [README.md](https://github.com/kruize/kruize-demos/blob/main/README.md) for reference.

Reply:
Added a detailed description